app/uk/gov/hmrc/gform/summary/SummaryRenderingService.scala

```diff
@@ -90,7 +90,11 @@ class SummaryRenderingService(
         HtmlSanitiser
           .sanitiseHtmlForPDF(
             summaryHtml,
-            document => HtmlSanitiser.acknowledgementPdf(document, submissionDetailsString, cache.formTemplate)))
+            document => {
+              document.title(s"Form Summary - ${cache.formTemplate.formName.value}")
```

Comment:
`Form Summary` shouldn't this be localised?

Reply:
Probably yes, thanks